### PR TITLE
add getAllValues to js related sdks

### PR DESCRIPTION
**Pull request by endret**

### CODE REVIEW

File: website/docs/sdk-reference/js-ssr.md [hunk 1]
```diff
@@ -286,6 +286,28 @@ let configCatClient = configcat.createClient("#YOUR-SDK-KEY#");
 const keys = await configCatClient.getAllKeysAsync();
 console.log(keys);
 ```
+
+## `getAllValues()`
+
+You can evaluate all keys with `getAllValues()` method.
```
Comment (laliconfigcat) [inline website/docs/sdk-reference/js-ssr.md:292]: maybe with *the*

File: website/docs/sdk-reference/js-ssr.md [hunk 2]
```diff
@@ -286,6 +286,28 @@ let configCatClient = configcat.createClient("#YOUR-SDK-KEY#");
...
+
+```js
+var configCatClient = configcat.createClient("#YOUR-SDK-KEY#");
```
Comment (laliconfigcat) [inline website/docs/sdk-reference/js-ssr.md:295]: var -> const

File: website/docs/sdk-reference/js-ssr.md [hunk 3]
```diff
@@ -286,6 +286,28 @@ let configCatClient = configcat.createClient("#YOUR-SDK-KEY#");
...
+    settingValues.forEach(i => console.log(i.settingKey + ' -> ' + i.settingValue));
+});
+```
+
+## `getAllValuesAsync()`
+
+You can evaluate all keys with `getAllValuesAsync()` method.
+
+```js
+let configCatClient = configcat.createClient("#YOUR-SDK-KEY#");
```
Comment (laliconfigcat) [inline website/docs/sdk-reference/js-ssr.md:306]: let -> const

**Review by laliconfigcat — CHANGES_REQUESTED** [review]

File: website/docs/sdk-reference/js-ssr.md (hunk 1 above)
Comment (sigewuzhere) [reply website/docs/sdk-reference/js-ssr.md:292]: `Evaluates and returns the values of all feature flags and settings. Passing a [User Object](link to user object page in docs) is optional.`

File: website/docs/sdk-reference/js-ssr.md [hunk 4]
```diff
@@ -286,6 +286,28 @@ let configCatClient = configcat.createClient("#YOUR-SDK-KEY#");
...
+
+## `getAllValues()`
+
+You can evaluate all keys with `getAllValues()` method.
+
+```js
+var configCatClient = configcat.createClient("#YOUR-SDK-KEY#");
+configCatClient.getAllValues(function(settingValues) {
+    settingValues.forEach(i => console.log(i.settingKey + ' -> ' + i.settingValue));
+});
```
Comment (sigewuzhere) [inline website/docs/sdk-reference/js-ssr.md:298]: I think, plus 1 example with a User Object would be useful here.

Comment (sigewuzhere) [reply website/docs/sdk-reference/js-ssr.md:298]: And in the rest of the example codes too.

**Review by sigewuzhere — CHANGES_REQUESTED** [review]

**Review by sigewuzhere — APPROVED** [review]

**Review by laliconfigcat — APPROVED** [review]

**Pull request merged**